templates/receive-azure-event-grid-events.json

```diff
@@ -0,0 +1,87 @@
+{
+    "id": "/providers/Microsoft.Logic/galleries/public/templates/71e35685-6dd9-4519-a947-5ef51a554817",
+    "type": "Microsoft.Logic/galleries/templates",
+    "name": "71e35685-6dd9-4519-a947-5ef51a554817",
+    "properties": {
+        "author": {
+            "displayName": "Tom Kerkhove"
+        },
+        "categoryNames": [
+            "enterprise_integration",
+            "productivity",
+            "general"
+        ],
+        "description": "Provide HTTP endpoint to accept and handle Azure Event Grid events with schema validation.",
```

Comment:
Would it make sense to include what version of Event Grid schema this is?

Reply:
How is this trigger different than just using the EventGrid kind?

Reply:
Because the current event grid trigger only works for built-in Azure Event Grid events, not with custom events/topics.

![image](https://user-images.githubusercontent.com/4345663/55533080-64fc4d00-56b0-11e9-8b05-10d3037f78d8.png)

![image](https://user-images.githubusercontent.com/4345663/55533115-8826fc80-56b0-11e9-8f10-1ffd2329332c.png)

Reply:
You're looking at the EventGrid Connector. Request triggers has 3 kinds; HTTP, SecurityCenterAlert, EventGrid. The EventGrid kind already has the outer EventGrid envelope schema built-in and allows you to the specify the schema for the specific event including custom events.
![BuiltInTriggers](https://user-images.githubusercontent.com/2703408/55574371-17d4a700-56c1-11e9-911e-73028c341d1a.jpg)

![RequestTriggerKinds](https://user-images.githubusercontent.com/2703408/55574387-215e0f00-56c1-11e9-86be-f1a86d2ff43e.jpg)


![RequestTriggerEventGridKind](https://user-images.githubusercontent.com/2703408/55574396-2622c300-56c1-11e9-8836-d393af859d3a.jpg)

Reply:
That's great - Didn't know about that, is it possible to surface this in the search for a connector?

Reply:
As you can see from your own screen shot, when you search for Event Grid the Request trigger shows up, and when you select it you can pick the "When an Event Grid resource event occurs" operation.

Reply:
It's not ideal but is good enough, thanks for the heads up!